operation/fhir-operation-bulkdata/src/main/java/org/linuxforhealth/fhir/operation/bulkdata/model/transformer/JobIdEncodingTransformer.java

```diff
@@ -50,86 +40,54 @@ public static JobIdEncodingTransformer getInstance() {
         return transformer;
     }
 
-    /*
-     * gets the tenant specific encryption key.
-     * @return
+    /**
+     * Create the tenant-specific Hashids object with a "salt" from fhir-server-config
+     * @return the Hashids object or null if no salt is configured in fhir-server-config
      */
-    private SecretKeySpec getJobIdEncryptionKey() {
-        String encryptionKey = ConfigurationFactory.getInstance().getCoreBatchIdEncryptionKey();
-        SecretKeySpec secretKey = null;
-
-        if (encryptionKey != null && !encryptionKey.isEmpty()) {
-            try {
-                byte[] keyBytes = encryptionKey.getBytes("UTF-8");
-                keyBytes = Arrays.copyOf(MessageDigest.getInstance("SHA-1").digest(keyBytes), 16);
-                secretKey = new SecretKeySpec(keyBytes, "AES");
-            } catch (Exception e) {
-                logger.log(Level.WARNING, "Fail to generate encryption key from config!", e);
-            }
-        }
+    private Hashids createHashids() {
+        String salt = ConfigurationFactory.getInstance().getCoreBatchIdEncodingKey();
```

Comment:
Shouldn't the tenantId also be included as part of the salt?

Reply:
we have tenant-level protection on the job stuff already (a job is for a single tenant so if the wrong tenant asks for it then they will be rejected).

however, I guess it wouldn't hurt to include the tenantid in the salt... I can add that if you want

Reply:
done